newIDE/app/src/SceneEditor/index.js

```diff
@@ -205,6 +205,12 @@ export default class SceneEditor extends Component {
     this.setState({ scenePropertiesDialogOpen: open });
   };
 
+  openObjectEditor = () => {
+    if(!this.instancesSelection.getSelectedInstances()[0]) {return};
+    const selectedObjectInstanceName = this.instancesSelection.getSelectedInstances()[0].getObjectName();
```

Comment:
nitpicking: this variable represents the "name" of the "object" associated to the "selected instance". I would then name the variable `selectedInstanceObjectName`.